code/world.dm

```diff
@@ -3,8 +3,9 @@
 
 /world
 	mob = /mob/dead/new_player
-	turf = /turf/open/space/basic
 	area = /area/space
+	//turf = /turf/open/space/basic 	//Ask Kryyto if you want the space back
```

Comment:
That's creating a lot of errors but it's the best way to change the baseturf of everything, might need some refont